GetFed/GetFed/Controllers/FoodEntryViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  FoodEntryViewController.swift
+//  GetFed
+//
+//  Created by Britney Smith on 12/4/18.
+//  Copyright © 2018 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class FoodEntryViewController: UIViewController {
+    
+    // MARK - Outlets
+    @IBOutlet var foodTextField: UITextField!
```

Comment:
In order to reduce the number of views you need to interact with, you could create a custom `UIView` subclass that contains a label and text field, along with API to interact with it. Could clean things up a bit.

Reply:
Turned into a Trello card!